content/docs/user-guide/pipelines/external-dependencies-and-outputs.md

```diff
@@ -0,0 +1,205 @@
+# External Dependencies and Outputs
+
+Sometimes you need to stream your data dependencies directly from their source
+locations outside your local <abbr>project</abbr>, or stream your data outputs
+directly to some external location, like cloud storage or HDFS.
+
+## How external dependencies work
```

Comment:
The entire external dependencies section is moved from https://dvc.org/doc/user-guide/data-management/importing-external-data#how-external-dependencies-work and otherwise is untouched except for taking out `import` examples since they are addressed in other guides and aren't directly relevant to pipeline dependencies.